utils/python/wait_for_tests.py

```diff
@@ -307,30 +307,31 @@ def parse_test_status(file_contents):
     rv = OrderedDict()
     test_name = None
     for line in file_contents.splitlines():
-        tokens = line.strip().split()
-        if (len(tokens)==0 or tokens[0] == COMMENT_STATUS):
+        line = line.strip()
+        tokens = line.split()
+        if line == "" or line.startswith(COMMENT_STATUS):
```

Comment:
Changed this back to startswith so that we don't have to worry about "COMMENT something" vs "COMMENT: something" vs "COMMENT  : something".